lib/inc/hocon/config.hpp

```diff
@@ -594,7 +599,34 @@ namespace hocon {
         virtual std::vector<shared_object> get_object_list(std::string const& path) const;
         virtual std::vector<shared_config> get_config_list(std::string const& path) const;
 
-        // TODO: memory and duration parsing
+        // TODO: memory parsing
+
+        /**
+         * Gets a value as a std::chrono::duration, set to represent milliseconds.
+         * @param path the path to the time value
+         * @return the value as a duration in milliseconds
+         */
+        virtual boost::chrono::milliseconds get_duration(std::string const& path) const;
+
+        /**
+         * Gets a value as a number of the specified units.
+         * @param path the path to the time value
+         * @param unit the units of the number returned
+         * @return the value converted to the requested units
+         */
+         virtual double get_duration(std::string const& path, time_unit unit) const;
```

Comment:
What is the correct way to handle this "overload"?

For the record, the Java only provides the integer version.